test/lolex-test.js

```diff
@@ -1222,6 +1222,22 @@ describe("lolex", function () {
             assert.isTrue(spy.called);
         });
 
+        it("should support clocks with start time", function () {
+            var startingPoint = new Date("2018-01-01").getTime();
+            this.clock = lolex.createClock(startingPoint);
```

Comment:
This is a stupidly high number (`1514764800000`), but if you do e.g. `200` instead the test will fail with `invactions` being equal to `5` instead of hanging forever